pkg/local_object_storage/blobstor/common/storage.go

```diff
@@ -23,3 +27,53 @@ type Storage interface {
 	Delete(DeletePrm) (DeleteRes, error)
 	Iterate(IteratePrm) (IterateRes, error)
 }
+
+// Copy copies all objects from source Storage into the destination one. If any
+// object cannot be stored, Copy immediately fails.
+func Copy(from, to Storage) error {
```

Comment:
how about `Copy(dst, src)` param order? (like the standard one)

Reply:
changes nothing since `Storage` doesn't provide `io` pritmitives

Reply:
not critical. i was just reading its usage and thought that it is a mistake cause i get used to `copy(dst, src)` but it is not a mistake so i decided to just describe my short confusion here

Reply:
`Copy` is well-known, so semantically following its order of arguments makes things better.